akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala

```diff
@@ -65,13 +67,16 @@ private[cluster] final class ClusterHeartbeatSender extends Actor with ActorLogg
   import cluster.settings._
   import context.dispatcher
 
-  // the failureDetector is only updated by this actor, but read from other places
-  val failureDetector = Cluster(context.system).failureDetector
+  val oldestNWatchExternally = cluster.settings.MonitoredByNrOfMembers
+  val failureDetector: FailureDetectorRegistry[Address] = Cluster(context.system).clusterTeamInternalFailureDerector
+
+  val filterinternalClusterMembers: Member ⇒ Boolean =
```

Comment:
this is the "old" heartbeat, so it should only do internally.
FIXME typo in name